### PR TITLE
lang fr - 20220227

### PR DESCRIPTION
**Pull request by admiralnlson**

### CODE REVIEW

**Comment by admiralnlson** [comment]
**tl;dr:** everything below is really minor, you can definitely skip it.

The following keys work for English but they did not work as well for French:
```
{
	"monarch": {
		"settings": {
			"enableSheets": {
				"all": "all card",
				"hand": "Hand",
				"deck": "Deck",
				"pile": "Pile",
				"card": "Card",
				"notification": "Set 🦋 Monarch as the default for {sheet} sheets."
			}
		}
	}
}
```
Indeed, in French (and other non-Germanic languages), you'd want to capitalize the word in the button label, yes (kind of standard in Web UI).
But within the notification sentence however, you'd want it in lowercase since capitalized nouns in the middle of a sentence aren't a thing.

Also I had to use a trick to make the "{sheet}" variable work for 'all'.

It's a minor thing really, so don't feel obliged to. But replacing the notification parametrized sentence with a non-parametrized key for each use case (5 total) would solve this.


Conversely, if you want, I think you can merge the following keys:
- transparentHand/Deck/Pile.name
- fadeHand/Deck/Pile.name
- enableSheets.hand/deck/pile
Since I believe they represent the same concept.

Finally, I translated 'My hand' as 'Main' rather than 'Ma main' since it's not necessarily always your hand you're looking at. Could be another player's on which you have Observer permissions.
Possibly, this is something you want to consider for the English version as well.

That's it, I'm done.
Again, feel free to ignore any of the above, it's really quite minor.

**Comment by zeel01** [comment]
> Indeed, in French (and other non-Germanic languages), you'd want to capitalize the word in the button label, yes (kind of standard in Web UI).
> But within the notification sentence however, you'd want it in lowercase since capitalized nouns in the middle of a sentence aren't a thing.

It's weird in English too, the use of these capitalized words is something of a programmer affectation - treating the names of classes as proper nouns. Because "Card" means "Foundry VTT Card Document" not "a playing card" in this context. It's probably not "correct" to do this in English, but it's always the way I do it. But in addition... I didn't want twice as many strings just for capitalization reasons.

>Conversely, if you want, I think you can merge the following keys b/c I believe they represent the same concept:
>
>- transparentHand/Deck/Pile.name
>- fadeHand/Deck/Pile.name
>- enableSheets.hand/deck/pile

Then here, we have a bit of a technical limitation. The way the code for the settings menu works, it's assumed that each of these keys exist. 

> Finally, I translated 'My hand' as 'Main' rather than 'Ma main' since it's not necessarily always your hand you're looking at. Could be another player's on which you have Observer permissions. Or if you're the GM you might be looking any hand really.
Possibly, this is something you want to consider for the English version as well.

So my plans are to eventually have multiple sheet options for each deck/pile/hand container. The "My Hand" sheet is the sheet intended to represent someone's personal hand. In the future there might be another sheet for hands intended for another purpose, such as "GM Hand" or "Mini Hand" or something. Essentially, "My Hand" is the "title" of the sheet as "Monarch" is the title of the module. Is it especially important? No. Just explaining the rationale.

File: lang/en.json [hunk 1]
```diff
@@ -45,11 +45,11 @@
 			},
 			"world": {
 				"name": "World Settings",
-				"hint": "Theses settings apply to all players."
+				"hint": "These settings apply to all players."
```
Comment (zeel01) [inline lang/en.json:48]: Thanks for fixing this typo 😆
